src/Polytopes/Polyhedron/properties.jl

```diff
@@ -298,22 +292,20 @@ julia> nvertices(C)
 nvertices(P::Polyhedron) = pm_polytope(P).N_VERTICES - nrays(P)
 
 
-"""
-    rays(as, P)
+@doc Markdown.doc"""
+    rays(as::Type{T} = Points, P::Polyhedron)
 
+Return minimal set of generators of the cone of unbounded directions of `P`
```

Comment:
```suggestion
Return a minimal set of generators of the cone of unbounded directions of `P`
```
Or is it unique? Then of course it should be "the"